docs/design/coreclr/botr/intro-to-clr.md

```diff
@@ -254,6 +254,6 @@ Phew!  The runtime does a lot! It has taken many pages just to describe _some_ o
 - [CoreCLR Repo Documentation](README.md)
 
 [clr]: http://msdn.microsoft.com/library/8bs2ecf4.aspx
-[ecma-spec]: ../project-docs/dotnet-standards.md
+[ecma-spec]: ../../../project/dotnet-standards.md
```

Comment:
FYI, just noticed some links like this could be simpler in form:  
(docs/project/dotnet-standards.md)

Reply:
it would be nice if the same linter flagged those. I'll not make another pass now.